pyface/ui/wx/action/menu_manager.py

```diff
@@ -133,7 +139,7 @@ def is_empty(self):
 
         return self.GetMenuItemCount() == 0
 
-    def refresh(self):
+    def refresh(self, event):
```

Comment:
I thought we were setting `event=None` so that we could still call `refresh()` without any args?

```suggestion
    def refresh(self, event=None):
```